packages/sheriff-create-config/tsconfig.json

```diff
@@ -1,12 +1,12 @@
 {
   "$schema": "https://json.schemastore.org/tsconfig",
-  "include": ["src", "eslint.config.ts"],
-  "exclude": ["dist", "build", "node_modules"],
   "extends": "tsconfig/base.json",
   "compilerOptions": {
     "module": "ESNext",
     "moduleResolution": "Bundler",
     "incremental": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "declaration": true,
```

Comment:
this package is only used as a cli. We don't expose code to consumers, therefore we don't need to expose the types with a declaration file.